Assets/Scripts/movement.cs

```diff
@@ -22,6 +28,69 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-        
+        if(!PauseMenu.isPaused){
```

Comment:
Do we need to reupdate after game is un-paused? This is a gap in my unity knowledge so let me know if we don't please!

Reply:
Yes, but that is handled in the PauseMenu script. This code just prevents movement inputs from doing anything while the game is paused, which could allow the player to still press wasd and move, even though game is paused.